src/libraries/System.IO.FileSystem/tests/File/GetSetAttributes.cs

```diff
@@ -30,10 +47,37 @@ public void GetAttributes_MissingAlternateDataStream_Windows(string streamName)
             Assert.Throws<FileNotFoundException>(() => GetAttributes(streamName));
         }
 
+        [Theory,
+         InlineData(":bar"),
+         InlineData(":bar:$DATA")]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void GetAttributes_MissingAlternateDataStream_Windows_SafeFileHandle(string streamName)
+        {
+            string path = CreateItem();
+            streamName = path + streamName;
+
+            Assert.Throws<FileNotFoundException>(() =>
```

Comment:
The other tests don't use this either, so I'll mark this as resolved. However, it might make sense to refactor this everywhere in the future.

@danmoseley Maybe this should be extracted into an extra issue. Feel free.